subsys/bluetooth/host/smp.c

```diff
@@ -1472,6 +1472,10 @@ static void smp_pairing_complete(struct bt_smp *smp, u8_t status)
 {
 	BT_DBG("status 0x%x", status);
 
+	if(bt_auth->pairing_complete) {
+		bt_auth->pairing_complete(smp->chan.chan.conn, status);
```

Comment:
I think I suggested passing the no-bonding/bonding information to the callback, so the app has some idea whether the pairing will be discarded after the connection goes away, or if the keys will be stored persistently. Wouldn't this be useful to have?

Reply:
Can you check the callback definition again? Also, for your convenience, the suggestions you made last time can be referred here: https://lists.zephyrproject.org/g/devel/topic/22117716#4763